src/participant/participant.ts

```diff
@@ -120,9 +123,44 @@ export default class ParticipantController {
       participantId: this._participant?.id,
     });
     this._participant.onDidReceiveFeedback(this.handleUserFeedback.bind(this));
+
+    this.subscribeToCopilotStatus();
+
     return this._participant;
   }
 
+  dispose(): void {
+    this._copilotStatusSubscription?.dispose();
+  }
+
+  subscribeToCopilotStatus(): void {
+    const updateCopilotStatusContext = (): void => {
+      const copilot = vscode.extensions.getExtension(COPILOT_EXTENSION_ID);
+      void vscode.commands.executeCommand(
+        'setContext',
+        'mdb.isCopilotActive',
+        copilot?.isActive === true
+      );
+      if (copilot !== undefined && !copilot.isActive) {
+        // TODO: This is a workaround for https://github.com/microsoft/vscode/issues/234426
+        // The onDidChange event only gets fired if extensions get added or removed but
+        // not when they are activated; so retry after a delay and check again.
+        setTimeout(() => {
```

Comment:
I could not think of a better way to handle this... we could get away with just a check in other places as it'd usually happen later than Copilot activation but because this happens at extension activation point, we don't have much guarantee over the order extensions get activated

Reply:
We already do it during the extension activation: https://github.com/mongodb-js/vscode/blob/main/src/mdbExtensionController.ts#L180-L185. Do we need to do it twice?

Reply:
On my Mac at least, it doesn't work when we set it on activation; at this point `copilot?.isActive` is `false` for me so the buttons don't show up 🫨

Reply:
Interesting, i have tried your branch without `subscribeToCopilotStatus` and it shows me icons in the tree, so the extension activation catches `copilot?.isActive` as `true`. Maybe you can try to move `setTimeout` to the extension activation method, so we retry there?

Reply:
yeah that did work for me but only with the set timeout. My guess is that for me Copilot always gets activated _after_ the MongoDB extension, maybe I just have more extensions than your setup or something else like that? 

It is always false otherwise at the moment of MongoDB extension activation.

Reply:
Let's place then this workaround with timeout inside the extension `activate` method https://github.com/mongodb-js/vscode/blob/main/src/mdbExtensionController.ts#L164 until we hear something from https://github.com/microsoft/vscode/issues/234426.